documentation/add/07-sqw_redesign.md

```diff
@@ -179,7 +200,7 @@ Other important methods, necessary for the analysis of different cuts from an sq
 |-----|---------|---|
 | *from_cur_to_targ_coord* | Transforms the pixels of the current image coordinate system into other coordinate system the requested cut is presented in. | |
 | *get_nrange* | return ranges of indexes of pixels which may contribute into current cut | |
-| *bin_pixels* | transform input pixels into the coordinate system, defined by the target projection and calculate the contribution from these pixels into the areas, defined by the target **axes_block**. (namely calculate number of pixels (*npix*) contributing into each cell of the **axes_block** and summ pixels signal and error data into *s* and *err* accumulator arrays)| |
+| *bin_pixels* | transform input pixels into the coordinate system, defined by the target projection and calculate the contribution from these pixels into the areas, defined by the target **axes_block**. (namely calculate number of pixels (*npix*) contributing into each cell of the **axes_block** and sum pixels signal and error data into *s* and *err* accumulator arrays)| |
```

Comment:
A lot of complicated info in this paragraph. I think it would be worth having a much more extended explanation.
While we keep this text, remove existing commas and add a comma after target projection

Reply:
hopefully done.